server/tests/services/discovery_test.py

```diff
@@ -27,45 +27,43 @@ def test_default_config(self):
     configure_endpoints_from_ingress(DEFAULT_INGRESS_RULES)
 
     for endpoint_path in get_all_endpoint_paths():
-      assert get_service_url(
-          endpoint_path) == f'https://api-root{endpoint_path}'
+      assert get_service_url(endpoint_path) == f'http://api-root{endpoint_path}'
 
   def test_configure_endpoints_from_ingress_1(self):
     """Tests simple ingress configuration."""
     configure_endpoints_from_ingress('tests/test_data/ingress/test1.yaml')
 
-    assert get_service_url('/query') == 'https://query-host:8080/query'
+    assert get_service_url('/query') == 'http://query-host:8080/query'
```

Comment:
@Fructokinase can you confirm this

Reply:
yes, that's correct. I forgot to fix the tests. Thanks!